npm/rake-examples.js

```diff
@@ -0,0 +1,44 @@
+var child_process = require('child_process');
+var fs = require('fs');
+var https = require('https');
+
+var Log = require('./log.js');
+var Build = require('./build.js');
+var build = new Build();
+var log = new Log();
+
+var examplesBuildDir = 'build/examples';
+if (!fs.existsSync(examplesBuildDir)) {
+  fs.mkdirSync(examplesBuildDir);
+}
+
+log.title('compile examples');
+stdout = child_process.execSync('opal --compile examples/asciidoctor_example.rb --dynamic-require warning --no-opal > ' + examplesBuildDir + '/asciidoctor_example.js');
```

Comment:
Is it possible to extract an "Opal compile" method that is shared across npm scripts? In the future, Opal may even provide an API...wouldn't that be nice.